### PR TITLE
Add the hidden alias `-e` for the `start` subcommand.

### PR DESCRIPTION
**Pull request by Abdiramen**
Adding the short flag alias gives us the hidden alias `-e` that can be consumed by third parties while maintaining `start` and `start -e`.

The following commands will work as expected:
```
wezterm -e vim
wezterm start vim
wezterm start -e vim
```
resolves: #2782

### CODE REVIEW

File: wezterm-gui-subcommands/src/lib.rs [hunk 1]
```diff
@@ -146,6 +146,7 @@ mod test {
 
 #[derive(Debug, Parser, Default, Clone)]
 #[command(trailing_var_arg = true)]
+#[command(short_flag_alias = 'e')]
```
Comment (wez) [inline wezterm-gui-subcommands/src/lib.rs:149]:
```suggestion
#[command(visible_short_flag_alias = 'e')]
```

seems to make it show up in the `--help` output:

```
; ./target/debug/wezterm --help
Wez's Terminal Emulator
http://github.com/wez/wezterm

Usage: wezterm [OPTIONS] [COMMAND]

Commands:
  start                  Start the GUI, optionally running an alternative program [aliases: -e]
```

**Review by wez — COMMENTED** [review]
Aha!  One minor suggestion so that it is discoverable via `--help`

File: wezterm-gui-subcommands/src/lib.rs (hunk 1 above)
Comment (Abdiramen) [reply wezterm-gui-subcommands/src/lib.rs:149]: The way clap appends that alias is slick!

**Comment by wez** [comment]
Closed by https://github.com/wez/wezterm/commit/e8886752e87c3240df4148828797459776abfa7f
(something weird happened when merging it, so closing this PR manunally)

**Comment by wez** [comment]
Thanks!

**Pull request closed without merging**